### ISSUE TITLE
ChevronDownGlyph rendered by Tabs reports prop type error on aria-hidden

### ISSUE THREAD

**Issue by austinbruch** [carbon-design-system/carbon#3417]
`ChevronDownGlyph` rendered by `Tabs` reports a prop type error on the `aria-hidden` prop. Expecting string, got boolean.

## What package(s) are you using?

- [X] `carbon-components`
- [X] `carbon-components-react`

## Detailed description

> Describe in detail the issue you're having.

Rendering `Tabs` is resulting in a prop type warning regarding the ChevronDownGlyph icon, specifically passing a boolean value to the `aria-hidden` prop when it should be a string.

> Is this issue related to a specific component?

`Tabs`

> What did you expect to happen? What happened instead? What would you like to
> see changed?

No prop type warning.

> What browser are you working in?

Chrome

> What version of the Carbon Design System are you using?

carbon-components 10.4.1
carbon-components-react 7.4.1

> What offering/product do you work on? Any pressing ship or release dates we
> should be aware of?

## Steps to reproduce the issue

Render `Tabs` component.

> Please create a reduced test case in CodeSandbox
>
> - Style and vanilla JS:
>   https://codesandbox.io/s/github/carbon-design-system/carbon/tree/master/packages/components/examples/codesandbox
> - React:
>   https://codesandbox.io/s/github/carbon-design-system/carbon/tree/master/packages/react/examples/codesandbox

https://codesandbox.io/s/codesandbox-h6wsh

## Additional information
Stack trace: 
```
Warning: Failed prop type: Invalid prop `aria-hidden` of type `boolean` supplied to `Icon`, expected `string`.
    in Icon (created by ForwardRef(ChevronDownGlyph))

    in ForwardRef(ChevronDownGlyph) (created by Tabs)

    in div (created by Tabs)

    in div (created by Tabs)

    in Tabs (created by App)

    in div (created by App)

    in App
```
